renderer/components/UI/TransactionFeeInput.js

```diff
@@ -18,8 +18,52 @@ import {
 } from './constants'
 
 const speeds = [TRANSACTION_SPEED_SLOW, TRANSACTION_SPEED_MEDIUM, TRANSACTION_SPEED_FAST]
-
-function TransactionFeeInput({ isQueryingFees, initialValue, label, field, formApi, fee }) {
+const speedMessageMap = [
+  {
+    threshold: 2,
+    messageKey: 'transaction_speed_description_fastest',
+  },
+  {
+    threshold: 4,
+    messageKey: 'transaction_speed_description_fast',
+  },
+  {
+    threshold: 12,
+    messageKey: 'transaction_speed_description_medium',
+  },
+  {
+    threshold: 100,
+    messageKey: 'transaction_speed_description_slow',
+  },
+  {
+    threshold: Infinity,
+    messageKey: 'transaction_speed_description_slowet',
+  },
+]
+function TransactionFeeInput({
+  lndTargetConfirmations,
+  isQueryingFees,
+  initialValue,
+  label,
+  field,
+  formApi,
+  fee,
+}) {
+  const value = formApi.getValue(field)
+  let targetConf
+  switch (value) {
+    case TRANSACTION_SPEED_SLOW:
+      targetConf = lndTargetConfirmations.slow
```

Comment:
I'd use map object in this case. Like
```
const CONF_MAP = {
[TRANSACTION_SPEED_SLOW]:lndTargetConfirmations.slow
...
}
const targetConf = CONF_MAP [value]
```
Imo that'd be more compact/readable. But this is up to you.